spec/NoteSpec.js

```diff
@@ -1,6 +1,6 @@
-function testCircleRadiusDefaultsTo10() {
-  var circle = new Circle();
-  assert.isTrue(circle.radius === 10);
+function testNoteCreation() {
```

Comment:
Console log each test's function for visibility (Probably worth adding this functionality in the test helper)